v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/schema/typemapping/UnifiedTypeMapper.java

```diff
@@ -96,6 +100,7 @@ public enum MapperType {
     MYSQL,
     POSTGRESQL,
     ORACLE,
-    SQLSERVER
+    SQLSERVER,
```

Comment:
It was not introduced in this PR, but why is there a mention of SQL server in this flow ?

Reply:
It was cover edge cases  to test checks on unmapped source type in the initial mapping [pull/1460](https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1460)

When we get to supporting `SQLSERVER` will add an artificial type like `UNSUPPORTED` here.